src/components/Datepicker/Datepicker.js

```diff
@@ -0,0 +1,185 @@
+    const fullWeekRange = moment.range(firstDayFirstWeek, lastDayLastWeek);
+
+    // Return filled week bins
+    return fullWeekRange.toArray('days').reduce((acc, val) => {
+      if (moment(val).day() % 7 === 0) { acc.push([]); }
+      acc[acc.length - 1].push([moment(val), moment(val).within(monthRange)]);
+      return acc;
+    }, [[]]).filter(bin => bin.length > 0);
+  }
+
+  renderYearOptions() {
+    const prevYears = moment(this.state.viewedDate).subtract(2, 'years');
+    const nextYears = moment(this.state.viewedDate).add(2, 'years');
+    return moment.range(prevYears, nextYears).toArray('years').map((momentYear) => {
+      const year = moment(momentYear).year();
+      return <option key={year} value={year}>{year}</option>;
+    });
+  }
+
+  renderYearPicker() {
+    return (
+      <div className={this.prefix(['shrink-none'])}>
+        <label className={this.prefix(['assistive-text'])} htmlFor="select-01">Pick a Year</label>
+        <div className={this.prefix(['slds-select_container'])}>
+          <select
+            value={this.state.viewedDate.year()}
+            className={this.prefix(['select'])}
+            onChange={this.viewYear}
+          >
+            {this.renderYearOptions()}
+          </select>
+        </div>
+      </div>
+    );
+  }
+
+  renderWeekHeader() {
+    return (
+      <thead>
+        <tr id="weekdays">
+          {moment.weekdaysShort().map(wd => <th key={wd} id={wd} tbody="col">
+            <abbr title={wd}>{wd}</abbr>
+          </th>
+          ) }
+        </tr>
+      </thead>
+    );
+  }
+
+  renderMonth() {
+    return (
+      <tbody>
+        {this.getMonthDays().map((week, weekIndex) => this.renderWeek(week, weekIndex))}
+      </tbody>
+    );
+  }
+
+  renderWeek(week, weekIndex) {
+    return (
+      <tr key={weekIndex}>
+        {week.map(([day, inRange], dayIndex) => this.renderDay(day, inRange, dayIndex))}
+      </tr>
+    );
+  }
+
+  renderDay(day, inRange, dayIndex) {
+    const classes = {
+      [this.prefix(['is-today'])]: day.isSame(moment(), 'day'),
+      [this.prefix(['disabled-text'])]: !inRange,
+      [this.prefix(['is-selected'])]: day.isSame(this.state.selectedDate, 'day'),
+    };
+    return (
+      <td key={dayIndex} className={classnames(classes)} headers={day.day()}>
+        <span className={this.prefix(['day'])} onClick={() => this.selectDay(day)}>
+          {day.date()}
+        </span>
+      </td>
+    );
+  }
+
+  render() {
+    return (
+      <div className={this.prefix(['datepicker', 'dropdown', 'dropdown--left'])}>
+        <div className={this.prefix(['datepicker__filter', 'grid'])}>
+          <div className={this.prefix(['datepicker__filter--month', 'grid', 'grid--align-spread', 'grow'])}>
+            <div className={this.prefix(['align-middle'])}>
+              <Button
+                icon
+                onClick={() => this.viewMonth(-1)}
+                className={this.prefix(['button', 'button--icon-container'])}
```

Comment:
you use our button, which should already have the `button` css class and `icon-container` is also a flavor afaik